src/main/java/edu/cursor/spring_hw1/conroller/AuthorController.java

```diff
@@ -0,0 +1,54 @@
+package edu.cursor.spring_hw1.conroller;
+
+import edu.cursor.spring_hw1.entities.Author;
+import edu.cursor.spring_hw1.entities.Book;
+import edu.cursor.spring_hw1.exceptions.AuthorNotFoundException;
+import edu.cursor.spring_hw1.repository.AuthorRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("library/author")
+public class AuthorController {
+    private final AuthorRepository authorRepository;
+
+    @Autowired
+    public AuthorController(AuthorRepository authorRepository) {
+        this.authorRepository = authorRepository;
+    }
+
+    @GetMapping
+    public String getAuthors() {
+        return authorRepository.getAuthors().toString();
+    }
+
+    @GetMapping("/booksOf/{id}")
```

Comment:
move this endpoint into `BookController`

should be:
`library/books?authorId={id}`

Reply:
If i move it into BookController I`ll forced to create AuthorRepositoryImpl in BookController, because Book doent know its author, is it ok?